Wallet test-recovery panic: runtime error: slice bounds out of range [:4] with length 3

Steps to reproduce the error:

$ rocketpool w t
Enter 24
Enter 'gunn'
panic: runtime error: slice bounds out of range [:4] with length 3
  
![image](https://user-images.githubusercontent.com/23104993/214849810-b6d8d0d7-4000-472c-9350-2ffce98ef6f8.png)

--- a/rocketpool-cli/wallet/bip39/mnemonic-validator.go
+++ b/rocketpool-cli/wallet/bip39/mnemonic-validator.go
@@ -33,7 +33,7 @@ func (mv *MnemonicValidator) AddWord(input string) error {
 		return errors.New("Invalid word")
 	}
 
-	if wordList[idx] != input && (len(input) < 4 || wordList[idx][:4] != input[:4]) {
+	if wordList[idx] != input && (len(input) < 4 || (len(wordList[idx]) < 4 || wordList[idx][:4] != input[:4])) {
 		return errors.New("Invalid word")
 	}
 

